cmd/vms.go

```diff
@@ -55,12 +68,54 @@ func (c VMsCmd) printDeployments(instTable InstanceTable) error {
 	return nil
 }
 
-func (c VMsCmd) printDeployment(dep boshdir.Deployment, instTable InstanceTable) error {
-	vmInfos, err := dep.VMInfos()
+type deploymentInfo struct {
+	depName string
+	vmInfos []boshdir.VMInfo
+}
+
+func parallelVMInfos(deployments []boshdir.Deployment, parallel int) (map[string][]boshdir.VMInfo, error) {
+	if parallel == 0 {
```

Comment:
This feels like surprising behavior - I would expect `--parallel=0` to mean "Don't parallelize this operation".  Could either set parallel to 1 if it's 0 or add a check in `printDeployments` that runs the normal `vmInfos` command if parallel is 0.